### ISSUE TITLE
Immutable payloads

### ISSUE THREAD

**Issue by totokaka** [ardoq/rxbeach#27]
To help users of RxBeach ensure functional style in their streams, it would be nice to have the payloads be completely immutable, by both recursively defining them as `readonly` and protecting them with `Object.freeze` and/or `Object.defineProperty(... writable: false})`.

This issue is a successor of #21, and already implemented in #25, but is blocked by implementation cost in Ardoq front.
